org.eclipse.xtext.xbase.ide/META-INF/MANIFEST.MF

```diff
@@ -3,12 +3,12 @@ Bundle-ManifestVersion: 2
 Bundle-Name: Xbase Generic IDE Services
 Bundle-Vendor: Eclipse Xtext
 Bundle-SymbolicName: org.eclipse.xtext.xbase.ide;singleton:=true
-Bundle-Version: 2.13.0.qualifier
+Bundle-Version: 2.14.0.qualifier
 Bundle-RequiredExecutionEnvironment: JavaSE-1.8
 Bundle-ActivationPolicy: lazy
 Require-Bundle: org.eclipse.xtext.ide,
  org.eclipse.xtext.xbase;visibility:=reexport,
- org.eclipse.xtext.xbase.lib;bundle-version="2.13.0",
+ org.eclipse.xtext.xbase.lib;bundle-version="2.14.0",
  org.objectweb.asm;bundle-version="[5.0.1,7.0.0)",
```

Comment:
Should we enforce 6.0.0 to avoid surprises?

Reply:
no i dont think so. we even release with this span.
maybe we should discuss this in one of the java 9 issues

Reply:
https://github.com/eclipse/xtext-core/issues/517